igneous/tasks/skeletonization/tasks.py

```diff
@@ -49,17 +49,17 @@ class SkeletonTask(RegisteredTask):
   Convert chunks of segmentation into chunked skeletons and point clouds.
   They will be merged in the stage 2 task SkeletonMergeTask.
   """
-  def __init__(self, cloudpath, shape, offset, mip, teasar_params, crop_zone, will_postprocess):
+  def __init__(self, cloudpath, shape, offset, mip, teasar_params, crop_zone, will_postprocess, info=None):
     super(SkeletonTask, self).__init__(cloudpath, shape, offset, mip, teasar_params, crop_zone, will_postprocess)
     self.cloudpath = cloudpath
     self.bounds = Bbox(offset, Vec(*shape) + Vec(*offset))
     self.mip = mip
     self.teasar_params = teasar_params
     self.crop_zone = crop_zone
     self.will_postprocess = will_postprocess
-
+    self.cloudinfo = info
   def execute(self):
-    vol = CloudVolume(self.cloudpath, mip=self.mip, cache=True, cdn_cache=False)
+    vol = CloudVolume(self.cloudpath, mip=self.mip, cache=True, info=self.cloudinfo, cdn_cache=False)
```

Comment:
I think the reason you're adding info here is because I set `cache=True`, which is probably generating warnings for you. That was just a convenience for me, it's not necessary. I would just remove `cache=True` and get rid of the info logic.